examples/basic/src/util/createKurt.ts

```diff
@@ -0,0 +1,29 @@
+import { Kurt, type KurtAdapter } from "@formula-monks/kurt"
+
+import { KurtOpenAI } from "@formula-monks/kurt-open-ai"
+import { OpenAI } from "openai"
+
+import { KurtVertexAI } from "@formula-monks/kurt-vertex-ai"
+import { VertexAI } from "@google-cloud/vertexai"
+
+const DEFAULT_MODEL = "gpt-4o"
+
+export const createKurt = (model: string | undefined): Kurt => {
```

Comment:
I think is more idiomatic to indicate parameter as optional with `?` rather than explicitly allow undefined.

Which also lets you declare a default value explicitly, minor, so up to you.

```ts
export const createKurt = (model = DEFAULT_MODEL): Kurt => {
  const adapter = findAdapter(model)
  if (!adapter) throw new Error(`Model ${model} is not supported.`)
  return new Kurt(adapter)
}

```

Reply:
Thanks for pointing out this missed opportunity!

Fixed in 79045e5.